runtime/adapters/src/lib.rs

```diff
@@ -611,6 +611,7 @@ where
 						Err(_) => return None,
 					}
 				}
+				PoolType::Aave => EmaPrice::from(1),
```

Comment:
why is the oracle price for any AAVE always `1`?

Reply:
because aave always swaps for this price

Reply:
given you are swapping underlying asset for atoken and vice versa